src/python_testing/TC_CNET_4_3.py

```diff
@@ -65,8 +65,7 @@ def steps_TC_CNET_4_3(self) -> list[TestStep]:
                      "TH reads InterfaceEnabled attribute from the DUT. Skip this and remaining steps if the connected network is not on the cluster currently being verified.",
                      "Verify that InterfaceEnabled attribute value is true"),
             TestStep(5, "TH reads LastNetworkingStatus attribute from the DUT",
-                     "LastNetworkingStatus attribute value will be within any one of the following values \
-                      Success, NetworkNotFound, OutOfRange, RegulatoryError, UnknownError, null"),
+                     "LastNetworkingStatus attribute value is Success"),
```

Comment:
Test pland does say `LastNetworkingStatus attribute value will be within any one of the following values Success, NetworkNotFound, OutOfRange, RegulatoryError, UnknownError, null` in https://github.com/CHIP-Specifications/chip-test-plans/blob/master/src/cluster/network_commissioning.adoc#tc-cnet-4-3-ethernet-verification-for-attributes-check-dut-server

Do we need to wait for a test plan update PR to merge?

Reply:
Thanks for the heads up! The correct link to updated test plan PR is https://github.com/CHIP-Specifications/chip-test-plans/pull/5103